src/diffusers/utils/loading_utils.py

```diff
@@ -135,3 +135,15 @@ def load_video(
         pil_images = convert_method(pil_images)
 
     return pil_images
+
+
+def get_module_from_name(module, tensor_name: str) -> Tuple[Any, str]:
```

Comment:
Add comment that this is copied from transformers.

Reply:
Done 3464d8373